inductiva/templating/manager.py

```diff
@@ -11,12 +13,53 @@ class TemplateManager:
     Template files contains variables that are replaced by
     values provided as keyword arguments to the render methods.
     The underlying template engine is the Jinja2 library.
+
+    Attributes:
+        TEMPLATE_EXTENSION (str): The file extension used by Jinja2 templates.
     """
 
-    @classmethod
-    def render_dir(cls,
-                   source_dir: types.PathOrStr,
-                   target_dir: types.PathOrStr,
+    TEMPLATE_EXTENSION = ".jinja"
+
+    def __init__(self, template_dir: str):
```

Comment:
If the TemplateManager is meant to be used only by accessing a class method (render_dir), let's try to avoid having a constructor that allows to create instances of it (what are the instances supposed to do?). That is quite confusing, I think.

Reply:
addressed in [e7fc2ce](https://github.com/inductiva/inductiva/pull/1504/commits/e7fc2ce0e82d140b76eac5b8ad93edc305a75da8)